assets/templates/oauth_clients_limit_exceeded.html

```diff
@@ -26,14 +26,14 @@ <h1 class="h4 h2-md mb-4 text-center">{{t "Too Many OAuth Clients Title" .Client
               {{t "Too Many OAuth Clients Available Actions"}}
             </p>
             {{if .PremiumURL}}
-            <a href="{{.PremiumURL}}" target="_blank" class="btn btn-primary btn-md-lg w-100 mb-2">
+            <a href="{{.PremiumURL}}" {{with .OpenLinksInNewTab}}target="_blank"{{end}} class="btn btn-primary btn-md-lg w-100 mb-2">
```

Comment:
I would prefer to keep `if`, and not the fancier but more complicated `with` in templates

Reply:
It's also optional with `if`